### PR TITLE
Searchbar search on carname

### PR DESCRIPTION
**Pull request by kboodi**
Disabled search button when input is empty
image

resolved the All Cars navigation issue that is raised by QA

Jira ticket links:
https://abhirams.atlassian.net/browse/P3ECWFC-96
https://abhirams.atlassian.net/browse/P3ECWFC-97

### CODE REVIEW

File: src/components/Searchbar/Searchbar.scss [hunk 1]
```diff
@@ -64,3 +64,8 @@ form .form-control:focus {
   background: $btn-color;
   color: $white;
 }
+
+// .btn-search:disabled {
+//   background-color: $lightGrey;
+//   color: $black;
+// }
```
Comment (pashedin) [inline src/components/Searchbar/Searchbar.scss]: unnecessary comments?

File: src/components/Searchbar/Searchbar.tsx [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
 function Searchbar() {
   const [searchWord, setText] = useState("");
+  const navigate = useNavigate();
+
   const handleTextChange = (event: any) => {
     setText(event.target.value);
   };
 
+  const handleSubmit = () => {
+    console.log(searchWord);
```
Comment (pashedin) [inline src/components/Searchbar/Searchbar.tsx]: log?

**Review by pashedin — CHANGES_REQUESTED** [review]
small things man
Also you'd need to resolve conflicts

File: src/components/Searchbar/Searchbar.tsx (hunk 2 above)
Comment (kboodi) [reply src/components/Searchbar/Searchbar.tsx]: removed

File: src/components/Searchbar/Searchbar.scss (hunk 1 above)
Comment (kboodi) [reply src/components/Searchbar/Searchbar.scss]: removed

**Review by pashedin — APPROVED** [review]

**Pull request merged**